client/view/base/BrowsersStat.css

```diff
@@ -26,8 +31,8 @@
   --a: 0;
```

Comment:
`--a` is too short name. It is OK for test and hacks, but real front-end we need something more readable.